config/sync/pathauto.pattern.artikel.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 3d20c1fa-d379-4d80-b40b-fad69336b45a
+langcode: da
+status: true
+dependencies:
+  module:
+    - node
+id: artikel
```

Comment:
Should this be `article` and should the file (then) be named `pathauto.pattern.article.yml`?